admin/AmazonAI-GeneralConfiguration.php

```diff
@@ -178,15 +180,36 @@ public function posttypes_gui() {
     }
 
     function tim_limitless_gui(){
-        if (  $this->common->is_tim_limitless_enabled() ) {
+        if($this->common->is_tim_limitless_enabled()){
             $checked                = ' checked ';
-        } else {
+        }else{
             $checked                = ' ';
         }
 
-        echo '<input type="checkbox" name="tim_limitless_enabled" id="tim_limitless_enabled" ' . esc_attr($checked) . ' > <p class="description"></p>';
+        echo '<input type="checkbox" name="tim_limitless_enabled" id="tim_limitless_enabled" ' . esc_attr($checked) .'onchange="toggleCheckbox(this)"' . '> <p class="description"></p>';
 
         echo '<p class="description">it is free</p>';
+        echo '<script>function toggleCheckbox(element)
+                 {
+                   if(element.checked){
+                       showDivs("none");
+                   }else {
+                       showDivs("block");
+                   }
+                 }
+                 function showDivs(display){
+                    var element = document.getElementById("amazon_polly_access_key_div");
+                    element.style.display = display;
+                    var element2 = document.getElementById("amazon_polly_secret_key_div");
+                    element2.style.display = display;
+                    var labels = document.getElementsByTagName("label");
+                    for(var i = 0; i < labels.length; i ++) {
+                        var attr = labels[i].getAttribute("for"); 
+                        if(attr == "amazon_polly_access_key" || attr == "amazon_polly_secret_key" ) {
```

Comment:
"amazon_polly_access_key" and "amazon_polly_secret_key" to consts, and use it in the HTML creation also

Reply:
i don't see a reason to go over all of the code and put all amazon strings in consts. i put all of our strings in a consts file.

Reply:
im not talking about amazon code. im talking about the code you added in this PR. these strings exists in 2 places - move them to consts